environment/src/main/java/com/sequenceiq/environment/credential/v1/converter/CredentialToCredentialV1ResponseConverter.java

```diff
@@ -108,6 +111,22 @@ public Credential convert(CredentialRequest source) {
         return credential;
     }
 
+    public Credential convert(ExtendedCloudCredential source) {
+        if (source == null) {
```

Comment:
you can use the `doIfNutNull` or `getIfNotNull` at method call of the converter

Reply:
thank, but I am fine with `if` :D